cloudinit/config/cc_grub_dpkg.py

```diff
@@ -130,35 +151,55 @@ def handle(
         log.debug("%s disabled by config grub_dpkg/enabled=%s", name, enabled)
         return
 
-    idevs = util.get_cfg_option_str(mycfg, "grub-pc/install_devices", None)
-    if idevs is None:
-        idevs = fetch_idevs(log)
+    dconf_sel = get_debconf_config(mycfg, log)
+    if dconf_sel is None:
```

Comment:
I don't see a way that this path could be hit. In `get_debconf_config()`, both branches return a truthy string.

We can remove this check, I think.